src/openms/source/QC/FragmentMassError.cpp

```diff
@@ -108,10 +109,118 @@ namespace OpenMS
     return theo_spectrum;
   }
 
+  void calculateFME_(PeptideIdentification& pep_id, const MSExperiment& exp, const QCBase::SpectraMap& map_to_spectrum, bool& print_warning, double tolerance, FragmentMassError::ToleranceUnit tolerance_unit, double& accumulator_ppm, UInt32& counter_ppm, WindowMower& window_mower_filter)
```

Comment:
make this a member function of the class not a free function in the OpenMS namespace
should probably be a static private member